### ISSUE TITLE
Repeated stdout: "Error: The server is already running. I don't want to start it twice."

### ISSUE THREAD

**Issue by sigboe** [alinmear/docker-conanexiles#8]
Hi, I am trying to get this docker to work. Using this command

```
docker run -d --name="ConanExiles" --net="bridge" -e TZ="Europe/Berlin" \
-e HOST_OS="unRAID" \
-e "CONANEXILES_ServerSettings_ServerSettings_AdminPassword"="xxx" \
-e "CONANEXILES_Engine_OnlineSubSystemSteam_ServerName"="xxx" \
-e "CONANEXILES_Engine_OnlineSubSystemSteam_ServerPassword"="xxx" \
-e "CONANEXILES_SERVER_TYPE"="pvp" -e "PUID"="99" -e "PGID"="100" \
-p 7777:7777/udp -p 7778:7778/udp -p 27015:27015/udp \
-v "/mnt/user/appdata/conanexiles":"/conanexiles":rw \
alinmear/docker-conanexiles
```

And the stdout shows this

```
linux32/
linux32/libstdc++.so.6
linux32/steamcmd
steamcmd.sh
steam.sh
/usr/lib/python2.7/dist-packages/supervisor/options.py:297: UserWarning: Supervisord is running as root and it is searching for its configuration file in default locations (including its current working directory); you probably want to specify a "-c" argument specifying an absolute path to a configuration file for improved security.
'Supervisord is running as root and it is searching '
2018-02-05 09:49:01,272 CRIT Supervisor running as root (no user in config file)
2018-02-05 09:49:01,272 WARN Included extra file "/etc/supervisor/conf.d/conanexiles.conf" during parsing
2018-02-05 09:49:01,282 INFO RPC interface 'supervisor' initialized
2018-02-05 09:49:01,282 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2018-02-05 09:49:01,282 INFO supervisord started with pid 69
2018-02-05 09:49:02,285 INFO spawned: 'conanexilesController' with pid 82
2018-02-05 09:49:03,287 INFO success: conanexilesController entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2018-02-05 09:49:17,851 INFO spawned: 'conanexilesServer' with pid 152
2018-02-05 09:49:19,514 INFO success: conanexilesServer entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
conanexilesServer: started
Error: The server is already running. I don't want to start it twice.
Error: The server is already running. I don't want to start it twice.
Error: The server is already running. I don't want to start it twice.
Error: The server is already running. I don't want to start it twice.
Error: The server is already running. I don't want to start it twice.
Error: The server is already running. I don't want to start it twice.
Error: The server is already running. I don't want to start it twice.
```

The error message that is repeated repeates, dont know, maybe every few minutes. I cant connect to the server, the UDP ports are port forwared in my router.

`docker version` displays this

```
Client:
 Version:      17.09.1-ce
 API version:  1.32
 Go version:   go1.8.3
 Git commit:   19e2cf6
 Built:        Thu Dec  7 22:21:47 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.09.1-ce
 API version:  1.32 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   19e2cf6
 Built:        Thu Dec  7 22:28:28 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

**Comment by sephallen** [comment]
It looks like you're using unRAID.
I had the same issue until I installed docker-compose (via nerdpack-cli, https://lime-technology.com/forums/topic/35866-unraid-6-nerdpack-cli-tools-iftop-iotop-screen-kbd-etc/) and then followed the example docker-compose.yaml template from the repo readme.

**Comment by alinmear** [comment]
@sigboe can you verfiy the post of @sephallen? Does this solve your problem?

**Comment by sigboe** [comment]
@sephallen @alinmear 

Hello, I am sorry I did not see the message earlier.

I have already tried unRAIDs docker compose, and no change.

However, I am closing this, because I can reproduce a similar issue in a Windows Server VM. I do have an enterprise firewall at home, but I dont see how that blocks it (I have openings), I will ask for help from a network engineer colleague.

But I do have outgoing traffic on a blacklist (everything goes out by default), and opened incomming UDP for the same ports as the portforwarding suggestion. 

Anyway, I will reopen if I can definitely show its not the firewall.
